fix: New activity toast appeared despite applying the activity stream  filter - EXO-61600

GouadriaHanen opened the pull request:
prior to this change,  the "New posts" chips  is displayed with all stream filter case.
after this change, the "New posts" chips should only be displayed if the filter applied is to "Any Activity"
If filtered to any other filter, then no chips at all

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

rdenarie committed:
fix: The thumbnail url contains twice the size in activity - EXO-61893 - meeds-io/meeds#580

Before this fix, the size of thumbnail is added in activity display, but is also provided by extensions like news. So, the size is added twice.
This fix remove the not needed size

GouadriaHanen committed:
fix: JS error when trying to edit a shared activity - EXO-61476 (#2209)

prior to this change, when trying to edit a shared activity a Js error is thrown " Cannot read properties of undefined (reading 'replaceAll' "and the edit drawer doesn't open
after this change, the edit drawer is opened

GouadriaHanen committed:
fix: New activity toast appeared despite applying the activity stream filter - EXO-61600

ahamdi reviewed (commented):
I am not sure but when we select spaces I host or my favorite spaces, shouldn't we get a notification for that ?

GouadriaHanen commented:
> I am not sure but when we select spaces I host or my favorite spaces, shouldn't we get a notification for that ?
The problem here is related to the space where the activity had been posted.
It may not part of my fav space or of my spaces that I host

ahamdi commented:
Will it be hard to get the information of the source of the activity (which space) and what if the activity belongs to it ?

azayati committed:
fix: Fix request for publisherRole feature flag activation launched for each users - EXO-61913 (#2214)


Prior to this change, for people page or space members page, the request for publisherRole feature activation is launched for each users which makes a lot of requests and decreases the performance of the corresponding portlets. After this commit, we will ensure to get the status of publisherRole feature flag only once into a global js variable eXo.env.portal.PublisherRolePromotionFeatureEnabled by calling directly featureService.isFeatureActiveForUser service instead of the corresponding rest service.

rdenarie committed:
fix: Avatar url is malformed when requesting in people page - EXO-61925 - meeds-io/meeds#586

Before this fix, some avatar are requested with url like this  :
.../avatar&size=65x65 when lastUpdated parameter is not present.
This fix add test to add the character ? or & before the size parameter in function of the presence or not of lastUpdate parameter

GouadriaHanen committed:
use v-show

ahamdi committed:
Merge branch 'feature/maintenance' into EXO-61600
